clients/python/cac_client/cac_client/client.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, config_dict):
 class CacClient:
     rust_lib = ctypes.CDLL(lib_path)
 
-    rust_lib.cac_new_client.argtypes = [ctypes.c_char_p, ctypes.c_int, ctypes.c_char_p]
+    rust_lib.cac_new_client.argtypes = [ctypes.c_char_p, ctypes.c_int, ctypes.c_char_p, ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int)]
```

Comment:
should the types be `c_int` or something like `c_long`, because the headers has type `long`, right ?

Reply:
No , it should be pointer only